haystack/document_stores/opensearch.py

```diff
@@ -165,6 +166,8 @@ def __init__(
                                index type and knn parameters). If `0`, training doesn't happen automatically but needs
                                to be triggered manually via the `train_index` method.
                                Default: `None`
+        :param batch_size: Number of Documents to index at once / Number of queries to execute at once. If you face
```

Comment:
The `write_documents` method of the `OpenSearchDocumentStore` also has a `batch_size` parameter with a default of 10_000. If we introduce a batch_size param in the init of the document store, we should also use `self.batch_size or batch_size` in `write_documents` and make the parameter `batch_size: Optional[int] = None` in the signature of the `write_documents` method of the `OpenSearchDocumentStore`.
Can we make the default 10_000 instead of 1_000 then to prevent a breaking change?

Reply:
I fixed `write_documents` and changed the default back to 10_000. I initially changed it to 1000 because I found [this](https://www.elastic.co/guide/en/elasticsearch/guide/current/bulk.html#_how_big_is_too_big) on the Elasticsearch documentation, but this information is probably outdated as it is for a former version.